docs/wiki/configurator/firmware-flasher-tab.md

```diff
@@ -155,8 +157,8 @@ For CRSF, ELRS, FPORT or GHST protocols, this is included by default to simplify
 | Dashboard                 | Enable external i2c Oled Display device (to be deprecated)                                                    |
 | EMFAT (AutoRun, Icon)     | Enable FAT emulation and icon for onboard flash or MSC                                                        |
 | ESC Serial (SK) Inc. 4way | Enable SimonK and ESC Serial support for flashing via 4way interface                                          |
-| Flash Storage             | Enable Blackbox Flash Storage                                                                                 |
-| FrSky OSD                 | Enable FrSky OSD support                                                                                      |
+| Flash Storage             | Auto-included in 4.5, enables Blackbox Flash Storage before 4.5                                               |
+| FrSky OSD                 | Auto-included in 4.5, enables FrSky OSD support before 4.5                                                    |
```

Comment:
-  `Flash Storage` aka `USE_FLASH` - is not automatically included - only if provided by the config.
- `FRSKYOSD` - needs to be included using Other Options

Reply:
What I'm saying is the option `Flash Storage` no longer shows up under `Other Options`.  It simply doesn't exist.  So I propose:
`Flash Storage - 4.4 and earlier only.  Enables Blackbox Flash Storage support.  In 4.5, is normally auto-included via the FC config file.  To manually add Flash to a 4.5 build, enter `FLASH` in Custom Defines.`
Likewise for FrSkyOSD:
`4.4 and earlier only. For 4.5, enter `FRSKYOSD` in `Custom Defines``